src/input.ts

```diff
@@ -63,6 +63,11 @@ class Input {
     return core.getInput('useSlim') !== 'false';
   }
 
+  getDockerImage(): string {
+    const dockerImage = core.getInput('renovate-docker-image');
+    return !!dockerImage && dockerImage !== '' ? dockerImage : 'renovate/renovate';
```

Comment:
```suggestion
    return !!dockerImage ? dockerImage : 'renovate/renovate';
```

The first condition will also handle empty string

Reply:
Automatic linting during commit simplifies it just by removing the `!!` in this way: https://github.com/renovatebot/github-action/pull/739/commits/ff1b06752ca13ce8d86f01d31c3f84cc2cd937b6